.travis.yml

```diff
@@ -7,17 +7,22 @@ sudo: false
 
 php:
   - 5.5
-  - 5.6
   - 7.0
 
+env:
+  matrix:
+    - PREFER="--prefer-lowest"
+    - PREFER=""
```

Comment:
I think something like:

```
matrix:
    include:
        - php: 5.6
          env: COMPOSER_FLAGS="--prefer-lowest"
        - php: 7
```

I think you may need to remove the `php` section in order to have 2
builds.

https://github.com/phpbench/phpbench/blob/master/.travis.yml#L28
https://github.com/symfony-cmf/routing-auto/blob/master/.travis.yml#L17

On Thu, Apr 14, 2016 at 10:25:49PM -0700, Johannes Wachter wrote:
>    In [1].travis.yml:
> 
>  >    - 7.0
>  >
>  > +env:
>  > +  matrix:
>  > +    - PREFER="--prefer-lowest"
>  > +    - PREFER=""
> 
>    how can i choose in which version the matrix exists?
> 
>    —
>    You are receiving this because you commented.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/massiveart/MassiveSearchBundle/pull/83#discussion_r59828280
>    2. https://github.com/massiveart/MassiveSearchBundle/pull/83/files/09ab3739145e8c3762221202225c858854ef7d7c#r59828280
>    3. https://github.com/massiveart/MassiveSearchBundle/pull/83/files/09ab3739145e8c3762221202225c858854ef7d7c#r59828280